src/main/scala/io/citrine/lolo/validation/Merit.scala

```diff
@@ -132,6 +134,64 @@ case object UncertaintyCorrelation extends Merit[Double] {
   }
 }
 
+/**
+  * Negative Log Probability Density (NLPD) in two dimensions
+  * NLPD is calculated for each point and the median value is returned.
+  *
+  * @param i index of the first label to be compared
+  * @param j index of the second label to be compared
+  * @param method method to calculate correlation coefficient
+  * @param observational whether or not to calculate the observational uncertainty
+  */
+case class NegativeLogProbabilityDensity2d(i: Int, j: Int, method: CorrelationMethod, observational: Boolean) extends Merit[Seq[Any]] {
+  override def evaluate(predictionResult: PredictionResult[Seq[Any]], actual: Seq[Seq[Any]], rng: Random): Double = {
+    val pvas = PredictedVsActualTwoDimensions.makePva2d(predictionResult, actual, i, j, method, observational)
+    val nlpd = NLPD(pvas)
+    StatsUtils.median(nlpd)
+  }
+
+  /**
+    * Bivariate negative log probability density calculation
+    * log(2 * pi * sigma_x * sigma_y * sqrt(1 - rho**2)) + r**2/2, where r is the Mahalanobis distance
+    * @param pvas sequence of bivariate predicted-vs-actual points
+    * @return sequence of NLPD value for each prediction
+    */
+  def NLPD(pvas: Seq[PredictedVsActualTwoDimensions]): Seq[Double] = {
+    pvas.map { pva =>
+      val term1 = math.log(2 * math.Pi * pva.sigmaX * pva.sigmaY * math.sqrt(1 - math.pow(pva.rho, 2.0)))
```

Comment:
`pva.sigmaX * pva.sigmaY * math.sqrt(1 - math.pow(pva.rho, 2.0)` this is det(Cov) in two dimensions?